revscoring/languages/english.py

```diff
@@ -35,15 +35,30 @@
     "yank", "yankee", "yid",
     "zipperhead"
 ])
+INFORMAL_WORDS = set([
+    'awesome', 'awesomest', 'awsome'
```

Comment:
It is missing a "," at the end of a few lines. It seems this causes some concatenation in Python (e.g. "awsomebla", "bullshitcant")